platform/platform-filter-delegate/src/main/java/org/codice/ddf/platform/filter/delegate/DelegateServletFilter.java

```diff
@@ -112,6 +97,155 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
     @Override
     public void init(FilterConfig filterConfig) throws ServletException {
         this.filterConfig = filterConfig;
+        hasBeenInitialized = true;
+
+        // now that the DelegateServletFilter has been initialized with a filterConfig, initialize all SecurityFilters that have not yet been initialized using that filterConfig
+        for (ServiceReference<SecurityFilter> securityFilterServiceReference : notYetInitializedSecurityFilterServiceReferences) {
```

Comment:
If we remove the notYetInitializedSecurityFilterServiceReferences we could just run through the sortedAndInitializedSecurityFilterServiceReferences list and intilialize those. If you did that you would probably want to rename the variable too. You could ignore my other comment about the hasBeenInitialized and set that to true at the end of this method and then use that variable in the do filter check above.